R/ospsuite-env.R

```diff
@@ -43,3 +43,22 @@ ospsuiteEnv$sensitivityAnalysisConfig$totalSensitivityThreshold <- 0.9
 
 # Indicates whether PK-Sim was loaded already. This will prevent unrequired initialization of the pksim assemblies
 ospsuiteEnv$isPKSimLoaded <- FALSE
+
+
+#' Get the value of a global ospsuite-R setting.
+#'
+#' @param settingName String name of the setting
+#'
+#' @return Value of the setting stored in ospsuiteEnv. If the setting does not exist, an error is thrown.
+#' @export
+#'
+#' @examples
+#' getOSPSuiteSetting("packageVersion")
+#' getOSPSuiteSetting("sensitivityAnalysisConfig")$totalSensitivityThreshold
+getOSPSuiteSetting <- function(settingName){

```

Comment:
@msevestre Added this method as proposed in https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/386 to remove mentioning of ospsuitEnv in the class documentation.

Reply:
ok good enough. We do not allow to write the values. We might want to do that for another PR